editor/editor_data.h

```diff
@@ -255,9 +254,9 @@ class EditorData {
 	void script_class_save_icon_paths();
 	void script_class_load_icon_paths();
 
-	Ref<Texture2D> extension_class_get_icon(const String &p_class) const;
+	String extension_class_get_icon_path(const String &p_class) const;
```

Comment:
What's this change for? How is returning path better if you load it anyway?

Reply:
In a previous iteration this made sense, and I forgot to leave it unchanged after I discarded the old idea. Fix coming in the next push.